src/App.tsx

```diff
@@ -1,4 +1,5 @@
 import { useState, useEffect, useRef } from 'react'
+import { DragDropContext, Droppable, Draggable, DropResult } from 'react-beautiful-dnd'
```

Comment:
We will need to add react-beautiful-dnd' to the dependencies in package.json

Reply:
The dragging and dropping isn't working for me. I'm getting these messages in the console:
react-beautiful-dnd - Unable to find draggable with id: 1726951967217

Reply:
Hmm it still isn't working. Is there a simpler library we can use for this? I'm imagining something that gives us a little handle to drag.